bin/rucio-replica-recoverer

```diff
@@ -244,7 +242,8 @@ Note that attempting the use the ``--vos`` argument when in single-VO mode will
     parser.add_argument('--vos', nargs='+', type=str, help='Optional list of VOs to consider. Only used in multi-VO mode.')
     parser.add_argument("--run-once", action="store_true", default=False, help='One iteration only.')
     parser.add_argument("--limit-suspicious-files-on-rse", action="store", default=5, help='Maximum number of suspicious replicas on an RSE before that RSE is considered problematic and the suspicious replicas on that RSE are declared "TEMPORARY_UNAVAILABLE". Default value is 5.')
-    parser.add_argument('--sleep-time', action="store", default=300, type=int, help='Concurrency control: thread sleep time after each chunk of work')
+    parser.add_argument('--sleep-time', action="store", default=3600, type=int, help='Concurrency control: thread sleep time after each chunk of work')
```

Comment:
I don't know the requirements. Is 1h maybe too much?

Reply:
We changed it to be 1 hour, because the number of executions per run is relatively high. We will have to make some adjustments to the default value of `--limit-suspicious-files-on-rse` to compensate for this change at some time in the future.

Reply:
Okay